node/src/manager/commands/rewind.rs

```diff
@@ -116,8 +128,19 @@ pub async fn run(
     println!("\nRewinding deployments");
     for deployment in &deployments {
         let loc = deployment.locator();
-        subgraph_store.rewind(loc.hash.clone(), block_ptr_to.clone())?;
-        println!("  ... rewound {}", loc);
+        let deployment_details = subgraph_store.load_deployment_by_id(loc.clone().into())?;
+
+        let block_ptr_to = block_ptr_to.clone().or(deployment_details.start_block);
+
+        if let Some(block_ptr) = block_ptr_to {
+            subgraph_store.rewind(loc.hash.clone(), block_ptr.clone())?;
+            println!("  ... rewound {}", loc);
+        } else {
+            println!(
+                "  ... skipping {} as it doesn't have a startBlock specified in manifest",
```

Comment:
Instead of skipping deployments for which there's no start block, I think I'd rather first cycle through all deployments and make sure all manifests have a start block. If not, we abort the entire operation.